packages/components/src/form-token-field/style.scss

```diff
@@ -1,7 +1,10 @@
-.components-form-token-field {
+.components-form-token-field__input-container {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: flex-start;
 	width: 100%;
 	margin: 0;
-	padding: 0;
+	padding: 4px;
```

Comment:
Yep, this would be a nice chore/enhancement. Not a blocker, I'd say. I would expect a separate future PR to explore more grid stuff in a holistic way to touch many of these things.